src/main/java/com/epam/module4/controller/reviewtask/UserController.java

```diff
@@ -0,0 +1,77 @@
+package com.epam.module4.controller.reviewtask;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.List;
+import java.util.ArrayList;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class UserController {
+
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/mydb";
+    private static final String USER = "root";
+    private static final String PASS = "password";
+
+    public void doGet(HttpServletRequest request, HttpServletResponse response) {
+        String userId = request.getParameter("userId");
+
+        try {
+            Class.forName("com.mysql.jdbc.Driver");
+            Connection conn = DriverManager.getConnection(DB_URL, USER, PASS);
+            Statement stmt = conn.createStatement();
+
+            String sql = "SELECT * FROM Users WHERE id = '" + userId + "'";
+            ResultSet rs = stmt.executeQuery(sql);
+
+            List<String> users = new ArrayList<>();
+            while (rs.next()) {
+                users.add(rs.getString("name"));
+            }
+
+            response.setContentType("application/json");
+            response.setStatus(200);
+
+            response.getWriter().println(users.toString());
+
+            rs.close();
+            stmt.close();
+            conn.close();
+
+        } catch (Exception e) {
+            System.out.println("Error: " + e.getMessage());
```

Comment:
We need some proper exception handling here - "log and swallow" is never a good bet. Handling should most likely be reflected in response.

On a side note, extracted message exception without the exception class often becomes meaningless, as the main information about what went wrong is in the exception class and the message can even be empty